AuthService/jwtproc/jwtProccessing.go

```diff
@@ -0,0 +1,46 @@
+package jwtproc
+
+import (
+	"authservice/models"
+	"github.com/golang-jwt/jwt/v5"
+	"time"
+)
+
+// ToDo: Extract or generate
+const jwtSecret = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmb28iOiJiYXIiLCJpc3MiOiJ0ZXN0IiwiYXVkIjoic2luZ2xlIn0.QAWg1vGvnqRuCFTMcPkjZljXHh8U3L_qUjszOtQbeaA"
+
+func BuildUserToken(user models.User, duration time.Duration, isRefresh bool) (string, error) {
+	claims := &UserClaims{
+		jwt.RegisteredClaims{
+			ExpiresAt: jwt.NewNumericDate(time.Now().Add(duration)),
+		}, user, isRefresh,
+	}
+	return buildToken(claims)
+}
+
+func buildToken(claims jwt.Claims) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	return token.SignedString([]byte(jwtSecret))
+}
+
+func DecoderTokenToUserClaims(tokenString string) (UserClaims, error) {
+	claims, err := decodeToken(tokenString, &UserClaims{})
+	cl := claims.(*UserClaims)
+	return *cl, err
+}
+
+func IsUserTokenValid(tokenString string) (bool, error) {
```

Comment:
nit: token can be a subject argument like `func (tokenString *string) IsUserTokenValid() (bool, error)
But this way is ok